### ISSUE TITLE
When the front-end uses ue5.3, PixelStreaming error occurs: chunk-4J4RTFCF.js? v=9d130ef5:472 Uncaught (in promise) TypeError: Cannot read properties of undefined (reading 'split')

### ISSUE THREAD

**Issue by AmbitionGan** [EpicGames/PixelStreamingInfrastructure#35]
**UE Version:**
E.g. UE 5.3

**Frontend Version:**
E.g. UE5.3

**Problem component**

click to start and you'll get an error: 6chunk-4J4RTFCF.js? v=9d130ef5:472 Uncaught (in promise) TypeError: Cannot read properties of undefined (reading 'split')!
**Description**

**Steps to Reproduce:**
Open the PixelStreamingI plugin in ue and open the use remote signaling server to implement streaming in the web browser, but click to strat at one point will report an error and will not display the streaming screen! You need to keep the ue editor open to display the screen, have you encountered this problem?

**Comment by callyourai** [comment]
I have experienced the same behaviour

**Comment by IndigoCloud6** [comment]
@AmbitionGan  @callyourai  I encountered the same problem, so I used F12 to trace the cause of the issue. I found that PreferredCodec was undefined, which made me suspect that my gpu  does not support the default video encoding method. Therefore, I changed the encoding method to VP8 via the command line, and the problem was resolved by using -PixelStreamingEncoderCodec=VP8.
![16a8bca1d50f51471723580658f7539](https://github.com/EpicGamesExt/PixelStreamingInfrastructure/assets/71397670/dfbefa37-ae3a-498e-8574-b984bd5c9b50)
![ca40438b36e409ebeaa03b1945fcf47](https://github.com/EpicGamesExt/PixelStreamingInfrastructure/assets/71397670/fdf5e635-ce0a-4000-834f-20f9895dee82)
